### PR TITLE
[res] Add quant test Quant_Conv_Mul_Add_002

### PR DESCRIPTION
**Pull request by jinevening**
This adds quant test Quant_Conv_Mul_Add_002.

ONE-DCO-1.0-Signed-off-by: Hyukjin Jeong <hj1.jeong@samsung.com>

---
Related to: https://github.com/Samsung/ONE/pull/8618

### CODE REVIEW

File: res/TensorFlowLiteRecipes/Quant_Conv_Mul_Add_002/test.recipe [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+operand {
+  name: "ofm_conv"
+  type: FLOAT32
+  shape { dim: 1 dim: 32 dim: 32 dim: 64 }
+}
+operand {
+  name: "mul_non_const"
+  type: FLOAT32
+  shape { dim: 1 dim: 1 dim: 1 dim: 64 }
+}
```
Comment (jinevening) [inline res/TensorFlowLiteRecipes/Quant_Conv_Mul_Add_002/test.recipe:35]: Note that Mul's input is non-const, so Quantize Op is inserted.

File: res/TensorFlowLiteRecipes/Quant_Conv_Mul_Add_002/test.rule [hunk 2]
```diff
@@ -0,0 +1,14 @@
+# To check mixed-precision quantization.
+# Conv, Mul: int16, Add: u8
+# Quantize Ops are inserted before Conv, after Mul, before Mul's non-const input.
+
+RULE    "VERIFY_FILE_FORMAT"      $(verify_file_format) '=' 1
+
+RULE    "CONV_INT16"              $(tensor_dtype ofm_conv) '=' INT16
+RULE    "WEIGHTS_INT16"           $(tensor_dtype filter) '=' INT16
+RULE    "BIAS_INT32"              $(tensor_dtype bias) '=' INT64
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Quant_Conv_Mul_Add_002/test.rule:9]: optional) `BIAS_INT64` like `Quant_Conv_Mul_Add_001` test ?

**Review by seanshpark — DISMISSED** [review]
LGTM

**Review by seanshpark — APPROVED** [review]
LGTM

**Review by llFreetimell — APPROVED** [review]
LGTM

**Pull request merged**